Classes/Domain/FormDataCleanup/FormDataCleanupService.php

```diff
@@ -0,0 +1,51 @@
+<?php
+declare(strict_types=1);
+
+namespace PunktDe\Form\Persistence\Domain\FormDataCleanup;
+
+/*
+ *  (c) 2021 punkt.de GmbH - Karlsruhe, Germany - https://punkt.de
+ *  All rights reserved.
+ */
+
+use Doctrine\ORM\ORMException;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Persistence\Exception\IllegalObjectTypeException;
+use Neos\Flow\Persistence\Exception\InvalidQueryException;
+use PunktDe\Form\Persistence\Domain\Model\FormData;
+use PunktDe\Form\Persistence\Domain\Repository\FormDataRepository;
+
+class FormDataCleanupService
```

Comment:
Could possibly be removed if a delete query is used.

Reply:
I would like to leave this service. I think with this service we provide an easier way to integrate this functionality into other code.